incubation-process.md

```diff
@@ -87,19 +87,15 @@ Projects at the Adopted stage fit into the overall architecture of the OpenRail
 
 ### Criteria
 
-*The criteria for stage 3 are still a draft*
-
-*Examples from CNCF incubation process, to be adopted*
-
 * Document that it is being used successfully in production by at least three independent adopters which, in the TC’s judgement, are of adequate quality and scope.
```

Comment:
What means "independent"? How much do we need to define it? How much is left to the judgement of the TC? Judgement should mostly be on the TC in the individual case.